internal/kube/kube.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and IronCore contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package kube
+
+import (
+	"fmt"
+	"os"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// CreateKubeConfig creates a Kubernetes client configuration
+func CreateKubeConfig() (*rest.Config, error) {
```

Comment:
Can we construct here a `client.Client` instead of a REST config? That will make the usage up the chain easier as well as you only have to do `c.Get(ctx, ....)`.

Reply:
true, I missed this one